UsersGuide/source/pdemodelica.rst

```diff
@@ -0,0 +1,75 @@
+PDEModelia1
```

Comment:
Please fix this typo.

Reply:
It is called PDEModelica1 to emphasize it is just for 1-D problems and to distinguish from the extension by Saldamli which is for 3-D problems. It was not used it consistently. I amended and force-pushed a fix now.

Reply:
But it doesn't say PDEModelica1...

Reply:
Sorry Martin I don't understand.

Reply:
```diff
-PDEModelia1
+PDEModelica1
```